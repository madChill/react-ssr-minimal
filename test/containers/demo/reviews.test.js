import React from 'react';
import renderer from 'react-test-renderer';

import Review from '../../../src/containers/demo/reviews';

test('testing Review component is rendered', () => {
    const component = renderer.create(<Review />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
