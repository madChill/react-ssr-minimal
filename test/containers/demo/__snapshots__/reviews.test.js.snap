// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Review component is rendered 1`] = `
<div>
  <img
    alt=""
    src="IMAGE_MOCK"
  />
  review111ssdd pa11geddd1=======+=== 11 ss
</div>
`;
